src/test/java/com/thealgorithms/datastructures/heaps/LeonardoHeapHelperTest.java

```diff
@@ -0,0 +1,74 @@
+package com.thealgorithms.datastructures.heaps;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.junit.jupiter.api.Test;
+
+// TODO: CHeck file name. Check if the file needs to be here or in bitwose operations.
+public class LeonardoHeapHelperTest {
+
+    @Test
+    public void testConsecutiveTreeIndicesForZero() {
+        ArrayList<Integer> arrayList = LeonardoHeapHelper.findConsecutiveTreeIndices(0);
+        ArrayList<Integer> expectedList = new ArrayList<>(Arrays.asList(-1, -1));
+
+        assertEquals(expectedList, arrayList);
+    }
+
+    @Test
+    public void testConsecutiveTreeIndicesForEleven() {
+        ArrayList<Integer> arrayList = LeonardoHeapHelper.findConsecutiveTreeIndices(11);
+        ArrayList<Integer> expectedList = new ArrayList<>(Arrays.asList(0, 1));
```

Comment:
Looks like use `List.of()` is more appropriate here:

`List<Integer> expectedList = new ArrayList<>(List.of(0, 1));`